### ISSUE TITLE
"Error: Evaluation failed: TypeError: window.Store.Conn.serialize is not a function" after update

### ISSUE THREAD

**Issue by Cephas369** [pedroslopez/whatsapp-web.js#999]
Code:

```
const client = new Client({
    puppeteer: { headless: true, channel: 'chrome' },
    ffmpegPath: "C:\\ffmpeg-2021-05-30-git-51f1194eda-full_build\\bin\\ffmpeg.exe",
    session: sessionData,
    clientId: "3425464523",
});

client.on('qr', qr => {
    qrcode.generate(qr, { small: true });
});

client.on('authenticated', (session) => {
    console.log('Authenticated!')
})


client.on('ready', async () => {
    console.log('Client is ready!');
});

client.initialize();

**Comment by Cephas369** [comment]
Solved after adding the fixes from #989

**Comment by pedroslopez** [comment]
Closing! Also, just in case, please don't apply patches manually and instead just update the library to the latest version. This will ensure it's easier for you to apply updates down the line.
